src/main/java/DatabaseInteractin/Server.java

```diff
@@ -0,0 +1,31 @@
+package DatabaseInteractin;
+
+import Service.ProductService;
+import io.grpc.ServerBuilder;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.logging.Logger;
+
+public class Server {
+  public static final Logger logger = Logger.getLogger(Server.class.getName());
+
+  public static void main(String[] args) throws IOException, InterruptedException, SQLException {
+    // String user = args[0];
+    // String password = args[1];
+
+    io.grpc.Server server = ServerBuilder.forPort(50051).addService(new ProductService()).build();
```

Comment:
Can we receive the port as a program argument too?

Reply:
done